### PR TITLE
docs(router): support net.src.* and net.dst.* in http route expression

### PR DESCRIPTION
**Pull request by chronolaw**
### Description

In PR https://github.com/Kong/kong/pull/11950, we add support for `net.src.*` and `net.dst.*` in http route exepression.

It will be released in Kong gateway 3.6.0.
 

### Testing instructions

Preview link: 

### Checklist 

- [x] Review label added 
- [ ] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/_src/gateway/key-concepts/routes/expressions.md [hunk 1]
```diff
@@ -63,16 +63,16 @@ The following table describes the available matching fields, as well as their as
...
+| `net.src.port`                        | `Int`      | ✅                           | ✅                             | The port number used by the client to connect.                                                                                                                                                                                                                                                                        |
+| `net.dst.ip`                          | `IpAddr`   | ✅                           | ✅                             | Listening IP address where {{site.base_gateway}} accepted the incoming connection.                                                                                                                                                                                                                                                      |
```
Comment (Guaris) [inline app/_src/gateway/key-concepts/routes/expressions.md:74]:
```suggestion
| `net.dst.ip`                          | `IpAddr`   | ✅                           | ✅                             | Listening IP address where {{site.base_gateway}} accepts the incoming connection.                                                                                                                                                                                                                                                      |
```

File: app/_src/gateway/key-concepts/routes/expressions.md [hunk 2]
```diff
@@ -63,16 +63,16 @@ The following table describes the available matching fields, as well as their as
...
+| `net.dst.port`                        | `Int`      | ✅                           | ✅                             | Listening port number where {{site.base_gateway}} accepted the incoming connection.                                                                                                                                                                                                                                                     |
```
Comment (Guaris) [inline app/_src/gateway/key-concepts/routes/expressions.md:75]:
```suggestion
| `net.dst.port`                        | `Int`      | ✅                           | ✅                             | Listening port number where {{site.base_gateway}} accepts the incoming connection.                                                                                                                                                                                                                                                     |
```

**Review by Guaris — CHANGES_REQUESTED** [review]
Small changes

**Comment by Guaris** [comment]
@chronolaw Can you point this at `release/gateway-3.6` please

**Comment by Guaris** [comment]
The conditional formatting I added may not render correctly until the branch is rebased over `release/gateway-3.6`

**Comment by chronolaw** [comment]
I will create a new one since it changes too many files.

**Comment by chronolaw** [comment]
replaced by #6765.

**Pull request closed without merging**